### ISSUE TITLE
Outdated content

### ISSUE THREAD

**Issue by EdCharbeneau** [software-architects/learn-blazor#25]
At the rate Blazor has changed over the last few previews, much of the content is out dated.

I've noticed community members warning newcomers not to visit the site.

What are the future plans for learn-blazor? It's constantly listed in search results, but fairly unreliable content at this point.

**Comment by rstropek** [comment]
I have to apologize that I don't have more time providing updates to *learn-blazor*. I have been doing Blazor work continously (e.g. [this presentation and sample](http://www.software-architects.com/devblog/2019/04/devone-blazor)) but I had not enough time to keep topics up to date. Additionally, I don't see much value given that Microsoft has Blazor docs available. When I created *learn-blazor*, there was no MS docs on that project.

Today, I have replaced many pages of *learn-blazor* with links to chapters in Microsoft docs and added statements that Microsoft docs are the preferred source of information about Blazor.
